SpatialGDK/Source/SpatialGDK/Public/Utils/SchemaDatabase.h

```diff
@@ -56,6 +56,16 @@ class SPATIALGDK_API USchemaDatabase : public UDataAsset
 		return SpatialConstants::INVALID_COMPONENT_ID;
 	}
 
+	uint32 GetComponentIdForClass(const UClass& Class) const
```

Comment:
is it worth making this parameterized by ESchemaComponentType also?

Reply:
not until we have a use case.